lesson17/app.js

```diff
@@ -1,145 +1,127 @@
-const url = "https://jsondata.okiba.me/v1/json/ngovH210301060343";
 const parent = document.getElementById("js-parent");
-const imagesFragment = document.createDocumentFragment();
 
-let imageLists;
-let pagination;
-
-const wait = (sec) => {
-  return new Promise((resolve) => {
-    setTimeout(resolve, sec * 1000);
-  });
+const sliderStatus = {
+  count: 0,
+  currentPagination: "",
+  imageLists: "",
 };
 
-async function getJsonData() {
-  try {
-    await wait(3);
-    const response = await fetch(url);
-    const jsonData = await response.json();
-    return jsonData;
-  } catch (err) {
-    throw Error(err);
-  } finally {
-    console.log("終了しました");
-  }
-}
-
-async function init() {
-  let data;
-  try {
-    data = await getJsonData();
-  } catch (e) {
-    console.error();
-  } finally {
-    console.log("終了しました");
-  }
-  if (data.length !== 0) {
-    createElements(data);
-  } else {
-    parent.innerHTML = "data is empty";
-  }
-}
-init();
-
-function createElements(data) {
-  createOfListImageItem(data.images);
-  createButton();
-  createPagination();
-}
+const renderSlider = ({ data }) => {
+  parent.appendChild(getImageList(data));
+  parent.after(getDirectionButtons());
+  parent.after(getPagination());
```

Comment:
IMO
I feel that `create**` would be more appropriate, but what do you think?
```suggestion
  parent.appendChild(createImageList(data));
  parent.after(createDirectionButtons());
  parent.after(createPagination());
```

Reply:
Thanks for the review!!

I used the name of the function here in reference to the previous comment [by Moriken-sensei.](https://github.com/443n0511/js-lesson/pull/33#discussion_r827773774)
This time we used get because we are getting the element created in the function.